src/app.hooks.js

```diff
@@ -118,6 +119,33 @@ const displayInternRequests = (level) => (context) => {
 	return context;
 };
 
+/**
+ * For errors without error code create server error with code 500.
+ * In production mode remove error stack and data.
+ * @param {context} context
+ */
+const errorHandler = (context) => {
+	if (context.error) {
+		if (context.error.hook) {
+			delete context.error.hook;
+		}
+		// statusCode is return by extern services / or mocks that use express res.status(myCodeNumber)
+		if (!context.error.code && !context.error.statusCode) {
```

Comment:
```suggestion
		if (!context.error.code __OR__? !context.error.statusCode) {
```

Reply:
If code do not exist AND statusCode also not exist then no error with code exist.